### ISSUE TITLE
BUG - backend-proxy-middleware can't read API Hub Key from the secure store

### ISSUE THREAD

**Issue by zdravko-georgiev** [SAP/open-ux-tools#544]
### Description
Trying to preview an application that uses an OData service from the SAP API Business Hub fails with 401 Error. 

### Steps to Reproduce
1. Generate a List Report application by using the SAP API Business Hub
2. Select as OData service, e.g. `Activity Type - Read (A2X)`
3. After generation install the `backend-proxy-middleware` by executing `npm i @sap-ux/backend-proxy-middleware`.
4. Add the `@sap-ux/backend-proxy-middleware` to the ui5 dependencies in the `package.json`.
5. Remove the `backend` configuration in the `fiori-tools-proxy`.
6. Add the `backend-proxy-middleware` to the `ui5.yaml` like this

```
  - name: backend-proxy-middleware
    afterMiddleware: compression
    configuration:
      backend:
        path: /s4hanacloud
        url: https://sandbox.api.sap.com
        apiHub: true

```
7. Execute `npm start`.

### Expected results
Data is loaded in the application

### Actual results
Data is not loaded in the application

### Screenshots
If applicable, add screenshots to help explain the problem.

#### Version/Components/Environment	
Add any other context about the problem here
OS:                      
- [x] Mac OS
- [x] Windows
- [ ] Other

### Problem
{describe the problem} 

### Fix
{describe the fix} 

### Why was it missed
{Some explanation why this issue might have been missed during normal development/testing cycle}  

### How can we avoid this
{if we don’t want to see this type of issues anymore what we should do to prevent}

**Comment by zdravko-georgiev** [comment]
One can use the following application to reproduce the issue [project1.zip](https://github.com/SAP/open-ux-tools/files/8956042/project1.zip)

**Comment by zdravko-georgiev** [comment]
@tobiasqueck @ullasholla FYI

**Comment by tobiasqueck** [comment]
I had a look at the problem and fixed a copy & paste error in #545 however that does not solve the problem completely. 

The main problem is that the store module expects that all entities are name `fiori/v2/NAME` so in the case of the `api-hub` it expects `fiori/v2/api-hub`, however, the Fiori generator creates an entry for `fiori/system/api-hub`. 
Furthermore, when we are serializing the store value, we always expect a JSON string, but the api-hub key is just a plain string, so even when I manually change the name in my keychain app, the return value is not a string but an object like 
```
{
    '0': 'S',
    '1': 'e',
    '2': 'c',
    '3': 'r',
    '4': 'e',
    '5': 't',
}
``` 

@ullasholla we need you for this one.

**Comment by zdravko-georgiev** [comment]
Fixed with https://github.com/SAP/open-ux-tools/pull/545
